bundles/io/org.eclipse.smarthome.io.rest.core/OSGI-INF/bindingresource.xml

```diff
@@ -9,10 +9,10 @@
 
 -->
 <scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" name="org.eclipse.smarthome.io.rest.core.internal.binding.BindingResource">
-   <implementation class="org.eclipse.smarthome.io.rest.core.binding.BindingResource"/>
+   <implementation class="org.eclipse.smarthome.io.rest.core.internal.binding.BindingResource"/>
    <service>
-      <provide interface="org.eclipse.smarthome.io.rest.core.binding.BindingResource"/>
       <provide interface="org.eclipse.smarthome.io.rest.RESTResource"/>
+      <provide interface="org.eclipse.smarthome.io.rest.core.internal.binding.BindingResource"/>

```

Comment:
Question: Does it actually make sense to keep these "provide" lines, even after moving the resources to an internal package? Imho the "provide RESTResource" should be enough, right?

Reply:
Good point - nope, they're pretty much useless now.